lib/synapse/service_watcher/base.rb

```diff
@@ -6,13 +6,15 @@ class BaseWatcher
     include Synapse::Logging
 
     LEADER_WARN_INTERVAL = 30
+    MAX_REVISION = 99999999999
```

Comment:
I think this is a premature optimization, let's just count up and if it's an issue in the future we can fix it.